akka-remote/src/main/scala/akka/remote/artery/ArteryTransport.scala

```diff
@@ -436,8 +433,14 @@ private[remote] class ArteryTransport(_system: ExtendedActorSystem, _provider: R
   private def startMediaDriver(): Unit = {
     if (settings.Advanced.EmbeddedMediaDriver) {
       val driverContext = new MediaDriver.Context
-      if (settings.Advanced.AeronDirectoryName.nonEmpty)
+      if (settings.Advanced.AeronDirectoryName.nonEmpty) {
         driverContext.aeronDirectoryName(settings.Advanced.AeronDirectoryName)
+      } else {
+        // create a random name but include the actor system name for easier debugging
+        val uniquePart = UUID.randomUUID().toString
+        val randomName = s"${CommonContext.AERON_DIR_PROP_DEFAULT}-${system.name}-$uniquePart"
```

Comment:
I wonder why our files are located in `/run/shm`. In Aeron sources I only see `/dev/shm`